src/pages/index.tsx

```diff
@@ -4,6 +4,9 @@ import { APP_THEMES } from "@/common/tools/tailwind/themes/appTheme";
 import { useTheme } from "next-themes";
 import { useState, useEffect, useCallback } from "react";
 import { PageHead } from "@/common/components/PageHead";
+import { Icon } from "@iconify-icon/react";
+import { CrossIcon } from "@/common/components/CustomIcon/CrossIcon";
```

Comment:
Think you forgot to remove `CrossIcon` here as it has not been defined

Reply:
Oh right, thanks for checking that!